kobo/apps/accounts/templates/socialaccount/login.html

```diff
@@ -36,7 +36,7 @@ <h1>{% trans "Log in with " %} {{ appname }}</h1>
   {% endif %}
   {% csrf_token %}
   <p class="or-go-back">
-    {% trans "or" %} <a href="javascript:history.back()">{% trans "go back" %}</a>
+    {% trans "or" %} <a href="/accounts/login">{% trans "go back" %}</a>
```

Comment:
Let's not hard-code the url and use `url` filter instead, i.e. `{% url 'account_login' %}`